mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/quota/CassandraCurrentQuotaManager.java

```diff
@@ -68,19 +68,23 @@ public CassandraCurrentQuotaManager(Session session) {
     }
 
     @Override
-    public void increase(QuotaRoot quotaRoot, long count, long size) throws MailboxException {
-        checkArguments(count, size);
-        session.execute(increaseStatement.bind(count, size, quotaRoot.getValue()));
+    public void increase(CurrentQuota currentQuota) {
```

Comment:
Well, the argument you provide is not  a stock (currentQuota) but a diff of that stock (a diff of CurrentQuota) => that would justify having a separate POJO IMO